modules/vulnerability-management-export-workloads.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_mod-docs-content-type: CONCEPT
+[id="vulnerability-management-export-workloads_{context}"]
+= Exporting workload vulnerabilities via the API
+
+[role="_abstract"]
+This section provides example use cases to export workload vulnerabilities using {product-title}. In this context workloads are composed of deployments and their associated images. The export makes use of the `/v1/export/vuln-mgmt/workloads` streaming API. It allows the combined export of deployments and images. The `images` payload contains the full vulnerability information. The output is streamed and has the schema:
++
+[source,json]
+----
+{
+  {"result": {"deployment": {...}, "images": [...]}},
+  {"result": {"deployment": {...}, "images": [...]}},
+  ...
+}
+----
+
+The following examples assume that these environment variables have been set:
```

Comment:
need space after this line so that the bullet points display properly
![Screenshot 2024-06-28 at 8 35 13 AM](https://github.com/openshift/openshift-docs/assets/96192119/14b53b1d-92f8-4a84-b940-391f55f504f7)